client/tests/e2e/specs/organization_information.spec.ts

```diff
@@ -1,52 +1,65 @@
-  await expect(usersContainer.locator('.user-revoked-header').locator('.user-revoked-header__title')).toHaveText('Revoked');
-  await expect(usersContainer.locator('.user-revoked-header').locator('.title-h4')).toHaveText('0');
-});
+import { expect, msTest, setSmallDisplay } from '@tests/e2e/helpers';
+
+for (const displaySize of ['small', 'large']) {
+  msTest(`Org info default state on ${displaySize} display`, async ({ organizationPage }) => {
+    const container = organizationPage.locator('.organization-page-content');
+    const configContainer = container.locator('.organization-info');
+    const usersContainer = container.locator('.organization-users');
+
+    if (displaySize === 'small') {
+      await setSmallDisplay(organizationPage);
+      await usersContainer.locator('#invitations-button').isVisible();
+      await expect(usersContainer.locator('.gradient-button-text').locator('.button-large')).toHaveText('pending invitation');
+      await usersContainer.locator('.user-invite-button').isVisible();
+      await usersContainer.locator('.card-header__button').click();
+    }
+
+    await expect(configContainer.locator('.info-list-item').locator('.info-list-item__title')).toHaveText([
+      'External profile',
+      'User limit (excluding users with External profile)',
+      'Server address',
+    ]);
+    await expect(configContainer.locator('.info-list-item').nth(0).locator('.info-list-item__value')).toHaveText(['Enabled']);
+    await expect(configContainer.locator('.info-list-item').nth(1).locator('.info-list-item__value')).toHaveText(['Unlimited']);
+    await expect(configContainer.locator('.info-list-item').nth(2).locator('.server-address-value__text')).toHaveText(
+      /^parsec3:\/\/.+$/,
+    );
+
+    await expect(usersContainer.locator('.users-list-item').nth(0).locator('.users-list-item__title')).toHaveText('3');
+    await expect(usersContainer.locator('.users-list-item').nth(0).locator('.users-list-item__description')).toHaveText('Active');
+    await expect(usersContainer.locator('.users-list-item').nth(1).locator('.users-list-item__title')).toHaveText('0');
+    await expect(usersContainer.locator('.users-list-item').nth(1).locator('.users-list-item__description')).toHaveText('Revoked');
+    await expect(usersContainer.locator('.users-list-item').nth(2).locator('.users-list-item__title')).toHaveText('0');
+    await expect(usersContainer.locator('.users-list-item').nth(2).locator('.users-list-item__description')).toHaveText('Frozen');
+    await expect(usersContainer.locator('.user-active-list').locator('.label-profile')).toHaveText(['Administrator', 'Member', 'External']);
+    await expect(usersContainer.locator('.user-active-list').locator('.user-active-list-item__value')).toHaveText(['1', '1', '1']);
+  });
+}
 
-msTest('Org info after one revocation', async ({ usersPage }) => {
-  const sidebarButtons = usersPage
-    .locator('.sidebar')
+msTest('Org info after one revocation', async ({ connected }) => {
```

Comment:
Use the fixtures `usersPage`, revoke a user, then switch to organization info. You can also check that it works in small display mode, but you can just add it at the end of this test, no need to create a new test.